Automattic/woocommerce-payments
# Remove platform checkout feature flag for upe

Fixes https://github.com/Automattic/woocommerce-payments/issues/3807

#### Changes proposed in this Pull Request
Remove platform checkout feature flagged codes for UPE checkout block.

#### Testing instructions
1. With "shortcode checkout", when you finished typing the email address in the checkout form, you should see an API request that goes out to check if there is a user in platform checkout. 
2. With "block" checkout, when you finished typing the email address in the checkout form, there is no API request going out.

-------------------


**Post merge**
